lib/datadog/core/remote/transport/http/api.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require_relative '../../../encoding'
+require_relative '../../../transport/http/api/map'
+
+# TODO: Decouple standard transport/http/api/instance
+#
+# Separate classes are needed because transport/http/traces includes
+# Trace::API::Instance which closes over and uses a single spec, which is
+# negotiated as either /v3 or /v4 for the whole API at the spec level, but we
+# need an independent toplevel path at the endpoint level.
+#
+# Separate classes are needed because of `include Trace::API::Instance`.
+#
+# Below should be:
+# require_relative '../../../../ddtrace/transport/http/api/spec'
```

Comment:
Is this comment relevant?  And any other comment that state what the actual require should look like 😄

Reply:
These comments were [already in place](https://github.com/DataDog/dd-trace-rb/blob/ba9fc95f49ec48fd28ca69ecec1a16a0024d3252/lib/datadog/core/transport/http/api.rb#L17). I just updated them to reflect this PR's changes. If we think they're superfluous, I'm happy to remove them.